pkg/resources/prometheus/configmap.go

```diff
@@ -392,7 +392,7 @@ scrape_configs:
   scheme: https
   tls_config:
 {{ .ApiserverTLSConfig | indent 4 }}
-  
+
```

Comment:
FYI, this tiny little change where I removed the trailing newlines caused the major formatting changes in the YAML fixtures. :grin: